src/components/Badge/index.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+
+const StyledBadge = styled(Badge)`
+  text-transform: none;
+`;
```

Comment:
Is this still needed?

Reply:
yup, we need override this property: https://github.com/mongodb/leafygreen-ui/blob/master/packages/badge/src/Badge.tsx#L27